src/solc_0.5/Land/erc721/LandBaseTokenV3.sol

```diff
@@ -611,6 +686,10 @@ contract LandBaseTokenV3 is ERC721BaseTokenV2 {
         quadId = layer + x + y * GRID_SIZE;
     }
 
+    /// @param size size of the quad
+    /// @param x x coordinate of the quad
+    /// @param y y coordinate of the quad
```

Comment:
can we simply modify it to?
 ` /// @param size of the quad`
 ` /// @param x coordinate of the quad`
 ` /// @param y coordinate of the quad`